examples/resources/vmp_custom_rule_set/resources.tf

```diff
@@ -0,0 +1,69 @@
+resource "vmp_waf_custom_rule" "custom_rule_1" {
+  account_number = "ca5db"
+  name           = "Custom Rule 1"
+  
+  directive {
+      sec_rule {
+          name = "Sec Rule 1"
+          action {
+              id = 66000000 
+              msg = "Invalid user agent."
+              t = ["NONE"]
```

Comment:
Might be best to rename `t` to `transformations` to make it more user friendly

Reply:
Updated sdk references to Transformations to match the latest version. Thanks.

Reply:
Hmm the .tf configuration still uses `t`, could you change it there too?

Reply:
Updated now. Thanks.